File: day_08/part1.js
const fs = require('fs');

let lines = fs.readFileSync('input', {encoding: 'utf8'}).split(/\r?\n/);

let strlen_encoded = 0;
let strlen_decoded = 0;

for (let n in lines) {
    line = lines[n];
    strlen_encoded += line.length;
    decoded = eval(line)
    strlen_decoded += decoded.length
}

console.log('Diff: ' + (strlen_encoded - strlen_decoded));

File: day_08/python/part1.py
strings = open('../input.txt').read().splitlines()

strlen_encoded = 0
strlen_decoded = 0

for string in strings:
    strlen_encoded += len(string)
    decoded = eval(string)
    strlen_decoded += len(decoded)

strlen_diff = strlen_encoded - strlen_decoded

print('Diff: {}'.format(strlen_diff))
